Src/DependencyCollector/Net45.Tests/DependencyCollector.Net45.Tests.csproj

```diff
@@ -28,8 +28,8 @@
     <RootNamespace>Microsoft.ApplicationInsights.DependencyCollector</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.ApplicationInsights, Version=2.9.0-beta1-build1608.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\packages\Microsoft.ApplicationInsights.2.9.0-beta1-build1608\lib\net45\Microsoft.ApplicationInsights.dll</HintPath>
+    <Reference Include="Microsoft.ApplicationInsights, Version=2.9.0-beta1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
this version number is obviously wrong, but i don't think it will block the build. 
Do a sanity check that the build and tests succeeded